src/main/config/robot.xml

```diff
@@ -3,12 +3,12 @@
 <robot>
 	<!-- pdp canId="0" / -->
 	<pigeon canId="50"
-               rotation="90.0" />

```

Comment:
This change should be in practicebot's robot.xml; the pigeon is off by 90 degress on the comp bot